index.html

```diff
@@ -9,16 +9,135 @@
 </head>
 <body>
     <header>
-        <div id="logo">ABDOU</div>
+        <div id="logo"><a href="#home">ABDOU SHOP</a></div>
         <div id="open-menu"><button id="open-menu-button" type="submit"><img src="img/header/open-menu.png" alt="open menu"></button></div>
-        <div class="search-input"><input type="search" name="search" id="search" placeholder="Search for items and brands"><button type="submit"><img src="img/header/search-icon.png" alt="search button"></button></div>
         <nav class="navbar">
             <div id="close-menu"><button id="close-menu-button" type="submit"><img src="img/header/close-menu.png" alt="close menu"></button></div>
             <ul class="navbar-links">
-                <li><a href="#home">HOME</a></li>
-                <li><a href="#about">ABOUT</a></li>
+                <li><a class="navbar-link" id="first-link" href="#home">HOME</a></li>
+                <li><a class="navbar-link" id="second-link" href="#services">SERVICES</a></li>
+                <li><a class="navbar-link" href="#about">ABOUT</a></li>
             </ul>
         </nav>
     </header>
+
+    <div class="hero" id="home">
+        <h1>DO YOU WANNA LOOK BEAUTIFUL ?</h1>
+        <p id="h1-supportingText">Find The Most Beautiful And wanted Clothes In The Horizon !</p>
+        <blockquote>"What you wear is how you present yourself to the world, especially today, when human contacts are so quick. Fashion is instant language." <br>- Miuccia Prada</blockquote>
+    </div>
+
```

Comment:
- [x] Background of this section looks good already but as per requirements you should follow the design from [behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) that means colors, font-family, background etc.. but you were free to personalize just contents.  kindly do that even on the About Page. :innocent: 🙏🏻 

- Original Design:

![bs](https://user-images.githubusercontent.com/44745855/132836775-ef27cc32-7245-474d-9dc3-cea8a091e864.png)

 - Your Page
 
![hom11](https://user-images.githubusercontent.com/79058364/168654485-81ce02ed-ccff-4a18-a619-c88f961ceaaf.png)

Reply:
resolved 👍